project/src/components/app/app.tsx

```diff
@@ -9,18 +9,17 @@ import {OfferType, ReviewType} from '../../types/offer';
 import FavoritesScreen from '../favorites-screen/favorites-screen';
 
 type AppScreenProps = {
-  rentOffer: number;
   offers: OfferType[];
   reviews: ReviewType[];
   nearbyOffers: OfferType[];
 }
-function App({rentOffer, offers, reviews, nearbyOffers}:AppScreenProps): JSX.Element {
+function App({ offers, reviews, nearbyOffers}:AppScreenProps): JSX.Element {
```

Comment:
```suggestion
function App(): JSX.Element {
```
тебе в апе эти пропы не нужно вообще больше. на всех страницах ты должен доставать все с помощью редакса